spark-mongodb/src/main/scala/com/stratio/datasource/mongodb/client/MongodbClientFactory.scala

```diff
@@ -43,7 +44,12 @@ object MongodbClientFactory {
   /**
    * Scheduler that close connections automatically when the timeout was expired
    */
-  private val actorSystem = ActorSystem()
+  private val actorSystem = ActorSystem("mongodbClientFactory", ConfigFactory.load(ConfigFactory.parseString(
+    """
```

Comment:
I think, the more concise, notation `ConfigFactory.parseString("akka.daemonic=on")` would fit better in this case.